app/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Theme.FeatureDetectAndroid" parent="android:Theme.Material.Light.NoActionBar">
+        <item name="android:statusBarColor">@color/purple_700</item>
```

Comment:
This is the only place where a color from `colors.xml` is used -- you can inline its value or even remove this line completely as you can control the status bar from Jetpack Compose too. So, the `style` block would look like this:

```xml
<style name="Theme.FeatureDetectAndroid" parent="android:Theme.Material.Light.NoActionBar" />
```

In fact, the whole file can be removed, but there are cases when you need to control app's theme like this, so I would recommend keep it.

Reply:
fixed